cwf/gateway/integ_tests/gy_enforcement_test.go

```diff
@@ -742,13 +742,10 @@ func TestGyCreditExhaustionRestrict(t *testing.T) {
 	assert.NoError(t, err)
 	tr.WaitForEnforcementStatsToSync()
 
-	// Check that UE mac flow was not removed and data was passed
-	verifyPolicyUsage(t, tr, ue.GetImsi(), "static-pass-all-ocs2", 0, 5*MegaBytes)
```

Comment:
at this point, sessiond should have already removed the flow `static-pass-all-ocs`